index.html

```diff
@@ -0,0 +1,4 @@
+<head>
+    <h>Hello Fullstack Program!</h>
+    <link rel="stylesheet" href="styles.css">
+</head>
```

Comment:
The `<head>` tag is not the same as `<header>`.

Also, you need to specify `<h1>` for titles. Just `<h>` doesn't exist.


```suggestion
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>HTML 5 Boilerplate</title>
    <link rel="stylesheet" href="styles.css">
  </head>
  <body>
  <header>
    <h1>Hello Fullstack Program!</h1>
  </header>
  </body>
</html>
```

Reply:
I already corrected the errors mentioned above and I also added some more content to the head element.